gspread/models.py

```diff
@@ -785,21 +801,15 @@ def get_all_records(
             return []
 
         keys = data[idx]
-
-        if numericise_ignore == ['all']:
-            values = data[idx + 1:]
-        else:
-            values = [
-                numericise_all(
-                    row,
-                    empty2zero,
-                    default_blank,
-                    allow_underscores_in_numeric_literals,
-                    numericise_ignore
-                )
-                for row in data[idx + 1:]
-            ]
-
+        values = [
```

Comment:
This overrides dc70dd9343629311ccde0df2ab4c72ec93cfbebe
Could you please rebase so we can keep dc70dd9343629311ccde0df2ab4c72ec93cfbebe  intact?